### ISSUE TITLE
Intervals query wildcard expansion parameter

### ISSUE THREAD

**Issue by prasad2kin** [elastic/elasticsearch#65505]
Lucene intervals wildcard query method has an additional parameter `expansion` to specify how many expansions to consider before throwing an error.

Can we get an additional parameter to supply that argument using the elastic Intervals-wildcard query?

Also, instead of throwing an error, is there a way to get terms with `maxExpansions` and performing the query? As for some use-cases, that would make more sense.

Please have a look at the thread in the discussion board: https://discuss.elastic.co/t/intervals-wildcard-128-expansion-limit/256390

**Comment by MysterionRise** [comment]
Agreed with @prasad2kin, it's better to expose this as parameter. Is there any motivation behind hardcoding 128 as maxExpansions?
